src/App.scss

```diff
@@ -1,8 +1,19 @@
-$primary-bgcolor-rgba: unset; // TODO: need to remove in src
-@import 'config/style/_overides.scss';
-@import 'node_modules/synapse-react-client/dist/template_style/Index.scss';
-@import './portal-components/csbc-home-page/style/all.scss';
-@import 'node_modules/synapse-react-client/dist/style/main.scss';
+@use 'variable_overrides';
+@use 'variables' as Portal;
+
+@use 'node_modules/synapse-react-client/dist/style/main.scss' with (
+  $primary-action-color: Portal.$primary-action-color, 
+  $secondary-action-color: Portal.$secondary-action-color,
+  $primary-bgcolor-rgba: unset,
```

Comment:
main primary-bgcolor-rgba not overridden by variables, but instead always unset?

Reply:
Before this PR, the first line in this file was
```scss
$primary-bgcolor-rgba: unset; // TODO: need to remove in src
```

We don't currently set a specific value for this in any of the portals. Here, I am maintaining whatever the current behavior is, as I haven't looked into what's actually happening here.